Update manage-folders-and-rules-feature.md (#13043)
MicrosoftDocs/microsoft-365-docs: microsoft-365/enterprise/manage-folders-and-rules-feature.md

  > 
  > - Message level actions Copy/Move will be blocked.

### Example 1

> To enable folders and Rules feature for Microsoft 365 Groups in Outlook you can use the below cmdlet

Reviewer: ```suggestion
To enable the **Folders and Rules** feature for Microsoft 365 Groups in Outlook, you can use the following cmdlet:
```

> To enable folders and Rules feature for Microsoft 365 Groups in Outlook you can use the below cmdlet
> 
> ```powershell
> Set-OrganizationConfig -IsGroupFoldersAndRulesEnabled $true
> ```
> 
> Once the feature is enabled, by default only owner of the group has the permission to create folder, rename folder, move and copy messages across folders.

Reviewer: ```suggestion

```powershell
Set-OrganizationConfig -IsGroupFoldersAndRulesEnabled $true
```
This example enables the folders and rules feature for Microsoft 365 Groups in Outlook.

Once the feature is enabled, by default, only the group's owner has permission to create folders, rename folders, and move and copy messages across folders.
```

> Once the feature is enabled, by default only owner of the group has the permission to create folder, rename folder, move and copy messages across folders.

Reviewer: ```suggestion
Once the feature is enabled, by default, only the group owner has permission to create folders, rename folders, and move and copy messages across folders.
```

   Default value: false

### Example 1

> To enable Enable member permission option you can use the below cmdlet.

Reviewer: ```suggestion
To enable the **Enable member permission** option, you can use the following cmdlet:
```

> 
> To enable Enable member permission option you can use the below cmdlet.
> 
> ```powershell
> Set-OrganizationConfig -IsGroupMemberAllowedToEditContent $true
> ```

Reviewer: ```suggestion

To enable the Enable member permission option, you can use the following cmdlet.

```powershell
Set-OrganizationConfig -IsGroupMemberAllowedToEditContent $true
```
```

> Once this is enabled, group owners can provide group members with the ability to create folders, rename folders, copy, move, and delete messages by navigating to the group from Outlook > **Settings** > **Edit Group** > and selecting the option **All members will be able to create, edit, move, copy, and delete mail folders and rules within the group**. Group level member permission is handled by group owners.

Reviewer: ```suggestion
Once this option is enabled, group owners can provide group members with the ability to create folders, rename folders, and copy, move, and delete messages by navigating to the group from Outlook > **Settings** > **Edit Group** > and selecting the option **All members will be able to create, edit, move, copy, and delete mail folders and rules within the group**. Group-level member permission is handled by group owners.
```

   Default value: false

### Example 1

> To block “Move” message capability you can use the below cmdlet

Reviewer: ```suggestion
To block the **Move message** capability, you can use the following cmdlet:
```

```powershell
Set-OrganizationConfig -BlockMoveMessagesForGroupFolders $true
```

> [!NOTE]

> > Creation of **Move** rule is also disabled when `BlockMoveMessagesForGroupFolders` is enabled.

Reviewer: ```suggestion
> Creation of the **Move** rule is also disabled when `BlockMoveMessagesForGroupFolders` is enabled.
```
